content/en/docs/collector/_index.md

```diff
@@ -72,11 +72,16 @@ fixes for critical bugs and security issues. See the
 [support policies](https://github.com/open-telemetry/opentelemetry-collector/blob/main/VERSIONING.md)
 for more details.
 
-{{% docs/latest-release collector-releases /%}}
+## Distributions and releases {#releases}
+
+For information about Collector distributions and releases, including the
+[latest release][], see [Distributions](distributions/).
```

Comment:
I'm on the fence about using just the title "Releases" vs what is shown here "Distributions and releases". I decided to propose the longer title here since the Collector section has a separate "Distributions" page.

Thoughts @tiffany76 et all?

Reply:
I think the longer title is better. It's more inclusive and helps the cross reference to Distributions make sense. But I'm not married to that opinion.